### PR TITLE
add a few information about airspeed sensor and usage of it

### PR DESCRIPTION
**Pull request by avcuenes**

### CODE REVIEW

File: tr/README.md [hunk 1]
```diff
@@ -4,40 +4,42 @@
 
 [![Sürümler](https://img.shields.io/badge/release-master-blue.svg)](https://github.com/PX4/PX4-Autopilot/releases) [![Tartışma](https://img.shields.io/badge/discuss-px4-ff69b4.svg)](https://discuss.px4.io//) [![Slack](../assets/site/slack.svg)](https://join.slack.com/t/px4/shared_invite/zt-si4xo5qs-R4baYFmMjlrT4rQK5yUnaA)
```
Comment (hamishwillee) [inline tr/README.md:5]: @avcuenes Thank you, but we cannot take translations done in github. Can you revert all this.
We'd love your translations if there are enough people working on them to make it worth the cost of building a dedicated version - but they would need to be done in crowdin.  http://docs.px4.io/main/en/contribute/translation.html

File: en/sensor/airspeed.md [hunk 2]
```diff
@@ -31,7 +31,7 @@ Additionally, the [Avionics Anonymous Air Data Computer](https://www.tindie.com/
 
 ### Enable Airspeed Sensors
 
-Unlike most other sensor drivers, the airspeed sensor drivers are not automatically started.
+Unlike most other sensor drivers, the airspeed sensor drivers are not automatically started.This feature is added after v1.13.0
```
Comment (hamishwillee) [inline en/sensor/airspeed.md:34]: Are you saying that airspeed sensors ARE started automatically after v1.13? 

I thought in main branch now we weren't auto-starting them?

Comment (avcuenes) [reply en/sensor/airspeed.md:34]: No, i say the opposite.I am saying, airspeed sensors started automatically before v1.13.0

After some development in v1.13.0 PX4 hasn't start automatically airspeed sensor.

Comment (avcuenes) [reply en/sensor/airspeed.md:34]: Just look at this link
`https://github.com/PX4/PX4-Autopilot/commit/3f172dbfa7b35e0be4ce7cb6544b33a35ebd9531`

File: tr/README.md (hunk 1 above)
Comment (avcuenes) [reply tr/README.md:5]: Thanks for your reply. I

File: en/sensor/airspeed.md [hunk 3]
```diff
@@ -31,7 +31,6 @@ Additionally, the [Avionics Anonymous Air Data Computer](https://www.tindie.com/
 
 ### Enable Airspeed Sensors
 
-Airspeed sensor drivers are not started automatically.
 Enable each type using its [corresponding parameter](../advanced_config/parameters.md):
```
Comment (hamishwillee) [inline en/sensor/airspeed.md:34]:
```suggestion
Airspeed sensor drivers are not started automatically.
Enable each type using its [corresponding parameter](../advanced_config/parameters.md):
```

**Comment by hamishwillee** [comment]
Thanks very much. I am closing this to continue the discussion in https://github.com/PX4/PX4-user_guide/pull/2014

**Pull request closed without merging**